frontend/encore/reactjs.rst

```diff
@@ -23,3 +23,83 @@ That's it! Your ``.js`` and ``.jsx`` files will now be transformed through
 ``babel-preset-react``.
 
 .. _`babel-preset-react`: https://babeljs.io/docs/plugins/preset-react/
+
+Building a component
+--------------------
+
+Now that Encore can manage any React instance, let's build a simple component
+which display ``Hello World from React !``.
+
+In order to do this, let's add a new ``react.js`` file inside the ``assets`` folder:
+
+.. code-block:: javascript
+
+    // assets/react.js
+    import React from "react";
+    import ReactDOM from "react-dom";
+
+    import { App } from './components/App.jsx';
+
+    ReactDOM.render(
+        <App name="Hello World from React !"/>,
+        document.getElementById("react")
+    );
```

Comment:
We should show `assets/components/App.jsx` first. And then say "Now it's time to render it!" and show the entry file. In the current order, I get confused because I see you import `App.jsx` and I don't know what that is yet :)